src/main/java/org/sonatype/repository/vgo/internal/proxy/VgoProxyFacetImpl.java

```diff
@@ -89,14 +89,15 @@ protected Content getCachedContent(final Context context) throws IOException {
   protected Content store(final Context context, final Content content) throws IOException {
     VgoAssetKind assetKind = context.getAttributes().require(VgoAssetKind.class);
     TokenMatcher.State matcherState = vgoPathUtils.matcherState(context);
-    VgoAttributes vgoAttributes = vgoPathUtils.getAttributesFromMatcherState(matcherState);
-    switch(assetKind) {
+
+    switch (assetKind) {
       case VGO_INFO:
       case VGO_MODULE:
       case VGO_PACKAGE:
+        VgoAttributes vgoAttributes = vgoPathUtils.getAttributesFromMatcherState(matcherState);
         return putComponent(vgoAttributes, content, vgoPathUtils.assetPath(matcherState), assetKind);
       case VGO_LIST:
```

Comment:
Question: This is spanning metadata right? I don't remember us doing a merge at the group level for this?

Reply:
The list is only on a component (module) level so no merging is necessary.